src/main/java/org/haffson/adventofcode/days/day02/Day02.java

```diff
@@ -75,4 +76,34 @@ private int calculateCheckSum(List<String> myStringList) {
         });
         return countSameLetterArray[0] * countSameLetterArray[1];
     }
+
+    private String calculateCommonLetters(List<String> myStringList) {
+        for (int i = 0; i < myStringList.size(); i++) {
```

Comment:
I think you can combine the for-loops like:

```suggestion
        for (int i = 0, j = i + 1; i <  myStringList.size() && j < myStringList.size(); i++, j++) {
```

Reply:
How would you change the body? this won´t work: 

private String calculateCommonLetters(List<String> myStringList) {
        for (int i = 0, j = i + 1; i < myStringList.size() && j < myStringList.size(); i++, j++) {
            String word1 = myStringList.get(i);
            String word2 = myStringList.get(j);
            int count = 0;
            int position = 0;
            String stringBuilder = compareWordsForEqualLetters(word1, word2, count, position);
            if (stringBuilder != null) return stringBuilder;
        }
        return null;
    }

Reply:
Not sure if that works.

I guess combining the loops results in an iteration through the matrix like...
```
x
 x
  x
```

... whereas the following is desired:

```
xxx
 xx
  x
```